pkgs/development/libraries/theft/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.3";
+  name = "theft-${version}";
+
+  src = fetchFromGitHub {
+    owner = "silentbicycle";
+    repo = "theft";
+    rev = "v${version}";
+    sha256 = "1ibh8np12lafnrsrvjbbzlyq45zq654939x0y22vdnc6s8dpbhw4";
+  };
+
+  preConfigure = "patchShebangs ./scripts/mk_bits_lut";
+  # doCheck = true;
+  
+  propagatedBuildInputs = [ ];
+
+  buildInputs = [ ];
+
+  installFlags = [ "PREFIX=$(out)" ];
```

Comment:
This should be passed to the configure phase automatically. Are you sure this is needed?

Reply:
This package does not use autoconf or any other configuration option.  The --prefix option is supplied during the configurePhase, but no corresponding specification is normally provided during the installPhase.  The Makefile for this project assumes the older-style convention of specifying PREFIX when running the install target, which is what the above is intended to accomplish.    This is necessary and works correctly in my testing of this derivation.